### PR TITLE
Add 4 cases of virt-v2v network

### PR DESCRIPTION
**Pull request by chloerh**
test result:

> JOB ID     : 8349bf05548986b520584bb1d44bf5ed67550127
> JOB LOG    : /root/avocado/job-results/job-2016-06-02T10.50-8349bf0/job.log
> TESTS      : 4
>  (1/4) type_specific.local.specific_kvm.positive_test.linux.network.multi_netcards.output_mode.rhev: PASS (835.10 s)
>  (2/4) type_specific.local.specific_kvm.positive_test.linux.network.virtio.output_mode.rhev: PASS (718.39 s)
>  (3/4) type_specific.local.specific_kvm.positive_test.linux.network.rtl8139.output_mode.rhev: PASS (707.81 s)
>  (4/4) type_specific.local.specific_kvm.positive_test.linux.network.e1000.output_mode.rhev: PASS (707.27 s)
> RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0
> JOB HTML   : /root/avocado/job-results/job-2016-06-02T10.50-8349bf0/html/results.html
> TIME       : 2968.57 s

### CODE REVIEW

**Comment by will-Do** [comment]
ACK

File: v2v/tests/src/specific_kvm.py [hunk 1]
```diff
@@ -42,7 +42,9 @@ def run(test, params, env):
     debug_kernel = 'debug_kernel' == checkpoint
     multi_kernel_list = ['multi_kernel', 'debug_kernel']
     backup_list = ['virtio_on', 'virtio_off', 'floppy', 'floppy_devmap',
-                   'fstab_cdrom', 'fstab_virtio', 'multi_disks', 'sata_disk']
+                   'fstab_cdrom', 'fstab_virtio', 'multi_disks', 'sata_disk'
```
Comment (waynesun09) [inline v2v/tests/src/specific_kvm.py]: comma missed after 'sata_disk'

File: v2v/tests/src/specific_kvm.py [hunk 2]
```diff
@@ -221,6 +223,46 @@ def change_disk_bus(dest):
...
+    def check_multi_netcards(mac_list, virsh_session_id):
+        """
+        Check if number and type of network cards meet expectation
+        """
+        virsh_instance = virsh.VirshPersistent(session_id=virsh_session_id)
+        vmxml = vm_xml.VMXML.new_from_inactive_dumpxml(
+                vm_name, virsh_instance=virsh_instance)
+        iflist = vmxml.get_iface_all()
+        logging.debug('MAC list before v2v: %s' % mac_list)
+        logging.debug('MAC list after  v2v: %s' % iflist.keys())
```
Comment (waynesun09) [inline v2v/tests/src/specific_kvm.py]: one redundancy space if you are not try to align the output

Comment (chloerh) [inline v2v/tests/src/specific_kvm.py]: Yes, the extra space is to make the 2 output lines clearer to read. And the missing comma is added, please review

File: v2v/tests/src/specific_kvm.py [hunk 3]
```diff
@@ -221,6 +223,46 @@ def change_disk_bus(dest):
...
+        for key in options:
+            line += ' --' + key + ' ' + options[key]
+        line += ' --current'
+        logging.debug(virsh.attach_interface(vm_name, option=line))
+
+    def check_multi_netcards(mac_list, virsh_session_id):
+        """
+        Check if number and type of network cards meet expectation
+        """
+        virsh_instance = virsh.VirshPersistent(session_id=virsh_session_id)
```
Comment (waynesun09) [inline v2v/tests/src/specific_kvm.py]: is VirshPersistent needed here for get inactive dumpxml? seems only need get xml like change_network_model

Comment (chloerh) [inline v2v/tests/src/specific_kvm.py]: I want to use get_iface_all() function to get all mac list, therefore a virsh instance is needed. Or is there another way to provide a virsh instance?

Comment (waynesun09) [inline v2v/tests/src/specific_kvm.py]: I don't think get_iface_all need a seperate virsh instance, the vmxml have default base.virsh session instance as I think

Comment (waynesun09) [inline v2v/tests/src/specific_kvm.py]: I don't realize it's for remote, then ACK for me

**Comment by will-Do** [comment]
Okay, merge this PR now

**Pull request merged**